### PR TITLE
rambox: 0.4.5 -> 0.5.2

### PR DESCRIPTION
**Pull request by phryneas**
###### Motivation for this change
version bump, new checksums.
also, the binary was renamed in the new download (from Rambox to rambox).

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/rambox/default.nix [hunk 1]
```diff
@@ -42,19 +42,19 @@ in stdenv.mkDerivation rec {
   ];
 
   installPhase = ''
-    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" Rambox
-    patchelf --set-rpath "$out/share/rambox:${stdenv.lib.makeLibraryPath deps}" Rambox
+    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" rambox
```
Comment (gnidorah) [inline pkgs/applications/networking/instant-messengers/rambox/default.nix]: @phryneas Thanks for being faster than me! :) You would also like to change ``exec = name;`` to ``exec = "rambox";`` at https://github.com/phryneas/nixpkgs/blob/947d862f92e1b5b1ce0be111968ef63d507d51cf/pkgs/applications/networking/instant-messengers/rambox/default.nix#L18 so menu item will work

**Review by ghost — CHANGES_REQUESTED** [review]
exec = name; to exec = "rambox"; at https://github.com/phryneas/nixpkgs/blob/947d862f92e1b5b1ce0be111968ef63d507d51cf/pkgs/applications/networking/instant-messengers/rambox/default.nix#L18 so menu item will work

File: pkgs/applications/networking/instant-messengers/rambox/default.nix (hunk 1 above)
Comment (phryneas) [inline pkgs/applications/networking/instant-messengers/rambox/default.nix]: will do that. and of course, I forgot to create a branch for the pull request, so I'll have to open a second pull request in a second :)

Comment (gnidorah) [inline pkgs/applications/networking/instant-messengers/rambox/default.nix]: @phryneas Nix people will surely take the change from your master as well

Comment (phryneas) [inline pkgs/applications/networking/instant-messengers/rambox/default.nix]: Yeah, noticed that a moment later, too.
I added the changed "exec" line.

**Review by ghost — APPROVED** [review]
Thanks!

**Review by ghost — CHANGES_REQUESTED** [review]
@phryneas 
Hi. There is now a new hotfix release 0.5.3 https://github.com/saenzramiro/rambox/releases/ which corrects the login issue some people may have on 0.5.2 if they were using 0.4.5 and lower prior https://github.com/saenzramiro/rambox/issues/622
Could you please update your pull request to bring 0.5.3? Or if you have no time, I may do this instead, but since I'm not a part of nixpkgs, you will need to close this PR, so I will open a new one for 0.5.3 instead

**Comment by phryneas** [comment]
@gnidorah: I won't be near a nixos machine until monday, so I'll close this to allow you to send a new PR.

**Comment by gnidorah** [comment]
@phryneas Done, thanks! I've preserved you as commit author ;-)

**Comment by phryneas** [comment]
Very cool, thanks ^^

**Pull request closed without merging**